package.json

```
@@ -33,7 +33,7 @@
     "body-parser": "1.20.3",
     "content-disposition": "0.5.4",
     "content-type": "~1.0.4",
-    "cookie": "0.6.0",
+    "cookie": "^0.7.1",
```

```suggestion
    "cookie": "0.7.1",
```

Ref: https://github.com/expressjs/express/pull/6017#discussion_r1791588098